WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
Do not pretend you know more about this than me, or that your source trumps the oldest and most detailed source of that belief system. **Absolutely not.** I told you that there is more to Antinomianism than your definition, and more importantly, it completely omits that they thought they could sin at-will, as long as they asked forgiveness first.

The Epistles, with whole chapters explicitly made to address Antinomianism, were contemporary accounts of Antinomianism and explicitly discussed the theological points that Antinomians argued concerning God's boundless Grace. They are the most accurate and oldest references to Antinomianism, and they trump any an all successive accounts of it.

Besides, they can already commit incest and take concubines, so it's obvious Christians hate them.